SportHub/src/modules/AdminHeader/styles/style.module.scss

```diff
@@ -11,24 +11,13 @@ select::-ms-expand {
 
   .searchbar {
     display: flex;
-    left: 18vw;
-    top: 0;
-    @include flex-center;
     background: var(white);
```

Comment:
white isn't a var. var(--white) maybe is. Because every custom var should start with --. So you can use only white and it would work.